grammar.js

```diff
@@ -87,8 +86,7 @@ const PREC = {
   // while() {}
   // for() {}
   // repeat {}
-  // NOTE: See `NOTE ON PREC.RIGHT` above
-  FUNCTION_OR_LOOP: { ASSOC: prec.right, RANK: 2 },
+  FUNCTION_OR_LOOP: { ASSOC: prec.left, RANK: 2 },
```

Comment:
Note - can fix `prec.right()` -> `prec.left()` now that we don't have an optional `body`